tools/webdriver/webdriver/bidi/modules/storage.py

```diff
@@ -51,18 +51,16 @@ class Storage(BidiModule):
 
     # TODO: extend with `filter`.
     @command
-    def get_cookies(self, partition: Optional[PartitionDescriptor] = None) -> Mapping[str, Any]:
-        params = {}
-        if partition is not None:
-            params["partition"] = partition
```

Comment:
Properties with `UNDEFINED` values are not sent, so we can simply drop them here.